src/DataTable/Table.js

```diff
@@ -64,11 +81,20 @@ class Table extends React.Component {
                     </tr>
                 </thead>
                 <tbody>
-                    {realRows.map((row, idx) =>
-                        <tr key={row[rowKeyColumn] || row.key || idx} className={row.className}>
-                            {columnChildren.map((child) => this.renderCell(child.props, row, idx))}
-                        </tr>
-                    )}
+                    {realRows.map((row, idx) => {
+                        const customRowProps = getCustomRowProps(row);
+
+                        return (
+                            <tr
+                                key={row[rowKeyColumn] || row.key || idx}
+                                style={customRowProps.style}
+                                className={customRowProps.className}
+                                onClick={customRowProps.onClick}
```

Comment:
I believe we can just pass everything without checks... Something like: `{ ...(row.mdlRowProps || {}) }`